en/flight_modes/README.md

```diff
@@ -51,15 +51,15 @@ Throttle Command is mapped direct to Motor Speed.
 
 "Assisted" flight modes are also user controlled but offer some level of "automatic" assistance to gain or restore controlled flight.
 
-- [Altitude](../flight_modes/altitude.md): More easily control vehicle altitude, and in particular reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold position/heading against wind.
+- [Altitude](../flight_modes/altitude.md): More easily control vehicle altitude, and in particular reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold the x and y position/heading against wind.

```

Comment:
Revert this. This clarification is less correct because if you expand it out to "hold the x and y position/hold the x and y heading" you can see the second case doesn't work. Also it won't hold the z position either :-)

Reply:
should I replace it with:
"will not attempt to hold the x, y position against wind."   ?

Reply:
No, I'd fully revert to "will not attempt to hold position/heading against wind."

Reply:
In altitude mode, as I understand it, it holds z position against wind but not x and y position. Don't you think it will be better for understanding if we mention which axes position this mode holds steady, in order to make this mode distinct from position control mode? 

I also don't understand the significance of just putting "heading" here, since heading means yaw, whereas in altitude mode all angles (roll, pitch, yaw) are treated like in manual mode so they can drift with the wind.

Reply:
> In altitude mode, as I understand it, it holds z position against wind but not x and y position. Don't you think it will be better for understanding if we mention which axes position this mode holds steady, in order to make this mode distinct from position control mode?

Not really. I think the text was clear enough. 

> I also don't understand the significance of just putting "heading" here, since heading means yaw, whereas in altitude mode all angles (roll, pitch, yaw) are treated like in manual mode so they can drift with the wind.

Essentially heading is part of the differentiation with position control mode. In altitude mode if you're heading in a particular direction wind can blow you off course. In position control mode the vehicle will attempt to maintain its course against wind.

Reply:
How about this? 

"More easily control vehicle altitude, and in particular reach and maintain a fixed altitude when the RC sticks are centered. The mode does not use GPS, and hence will not attempt to hold the horizontal position (position in the x and y plane of the world frame) against wind. If you're heading the vehicle in a particular direction wind can blow you off course, where as in position control mode the vehicle will attempt to maintain its course against wind."

To me, position means x, y, z position. In Altitude control mode, when the sticks are centered the current z position/altitude is held constant against the wind. This is what I confirmed from the px4 team. But then I got confused when I read that the position is not maintained against the wind. Here, shouldn't we specify that the "horizontal" position is not held against the wind?

Reply:
Too wordy. I'll compromise: 

> More easily control vehicle altitude, and in particular reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold the vehicle x and y position or course against wind.